update-swagger-ui.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# This script copies files from `swagger-ui-dist` module into `api-explorer`
```

Comment:
Shell/bash scripts won't work on Windows, could you please rewrite this in Node.js?

Here are few modules you may find useful:

 - [rimraf](https://www.npmjs.com/search?q=rimraf) instead of `rm -rf` (if needed) 
 - [cpr](https://www.npmjs.com/package/cpr) instead of `cp`
 - [fs-extra](https://www.npmjs.com/package/fs-extra) provides both copy & remove
 - [shelljs](https://www.npmjs.com/package/shelljs) (similar to fs-extra)